style.css

```diff
@@ -0,0 +1,17 @@
+section {
+    height: 100vh;
+}
+
+nav{
+    position: fixed;
+    width: 100%;
+    top: 0%;
+    left: 0;
+    
+}
+
+@media screen and (max-width: 600px) {
+    h1 {
+        color: red;
```

Comment:
I think the unused space should be removed for cleaner code.

Reply:
Great job with your styling but i will recommend you remove the unnecessary space in your css code. By doing so you will make your code cleaner and easier to read and you will also make it easy for you and collaborator to debug.